src/Servers/Connections.Abstractions/src/Features/DefaultConnectionLifetimeNotificationFeature.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading;
+using Microsoft.AspNetCore.Http.Features;
+
+namespace Microsoft.AspNetCore.Connections.Features
+{
+    /// <summary>
+    /// Default implementation of <see cref="IConnectionLifetimeNotificationFeature"/>.
+    /// </summary>
+    public class DefaultConnectionLifetimeNotificationFeature : IConnectionLifetimeNotificationFeature
+    {
+        private readonly IHttpResponseFeature? _httpResponseFeature;
+
+        public DefaultConnectionLifetimeNotificationFeature(IHttpResponseFeature? httpResponseFeature)
```

Comment:
This isn't going to work once you rebase, Connections.Abstractions no longer depends on Http.Features, it depends on Extensions.Features. You'll have to move this class into the `Http` assembly where it's used.